data_source.go

```diff
@@ -236,6 +238,20 @@ func (ds *AnySource) Running() bool {
 	}
 }
 
+// Signed returns a per-channel value: whether data are signed ints.
+func (ds *AnySource) Signed() []bool {
+	return make([]bool, ds.nchan)
+}
+
+// VoltsPerArb returns a per-channel value scaling raw into volts.
+func (ds *AnySource) VoltsPerArb() []float32 {
```

Comment:
This seems incomplete, the Abaco source isn't even going to have volts per arb, it has phase per arb.

The simulated sources also don't have volts per arb, they just have arbs.

Reply:
This is a good point, but I considered the more correct "physical-units-per-arb" to be hopelessly confusing. Let's call it Volts-per-arb for now, and we can come up with a more sensible name when the Abaco or Roach is online. Maybe `physUnitsPerArb` is totally fine, but I'll make the change when the time comes.